src/hw10/Shopper.java

```diff
@@ -0,0 +1,71 @@
+package src.hw10;
+
+public class Shopper {
+    private Store store;
+    private Product[] cart;
+    private double cash;
+
+    public Shopper(Store store, int cartCapacity, double cash) {
```

Comment:
At the beggining the shopper doesn't know the store. 
You will need a constructor with cash only.
Also a function if you want to change or to go to a store: goToStore(Store store, int cartCapacity)